src/man/sssd.conf.5.xml

```diff
@@ -4005,6 +4007,36 @@ subdomain_inherit = ldap_purge_cache_timeout
                             should be comma-separated, such as
                             <quote>enable:passkey, enable:smartcard</quote>
                         </para>
+
+                        <para>
+                            The following table shows which local authentication
```

Comment:
Hi,

I would like to suggest to drop `local` here because as long as SSSD is online the given authentication method will be done online not local with `local_auth_policy = match`. Only when offline the method will be handled locally.

bye,
Sumit

Reply:
Fixed and rebased. What about the `local` mentioned above ?

>                             supported by the backend. With this option additional
>                             methods can be enabled which are evaluated and checked
>                             locally.

Reply:
Hi,

I think at this point it is ok because if this option is set to something other than the default `match` it will enable a method which will then be evaluated locally.

bye,
Sumit